File: Application/Services/Operations/Outsourced/Dtos/CollectDeliverDto.cs
using System;
using Application.Services.Operations.Main.Partners.Dtos;
using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.Outsourced.Dtos
{
    public class CollectDeliverDto : RootBaseDto
    {
        public string ContactName { get; set; }
        public DateTime Start { get; set; }
        public decimal Price { get; set; }
        public DateTime WasPaid { get; set; }
        public DateTime Collect { get; set; }
        public DateTime Deliver { get; set; }
        public DateTime Other { get; set; }
        public string KindTransport { get; set; }
        public string TaskOverView { get; set; }
        public int? TransporterId { get; set; }
        public PartnerDto Transporter { get; set; }
        public BillingFromDto BillingFrom { get; set; }
        public DestinyDto Destiny { get; set; }
    }
}

File: Views.old/src/components/out-sourced/collect-deliver/dto/collect-deliver-dto.ts
import { MyUser } from "src/components/authentication/dto/my-user";
import { PartnerDto } from "src/components/main/partner/commons-components/dtos/partner-dto";
import { CompanyDto } from "src/shared/entities-dtos/company-dto";
import { BillingFromDto } from "./billing-from-dto";
import { DestinyDto } from "./destiny-dto";


export class CollectDeliverDto {
  id: number;
  companyId: number;
  company: CompanyDto;
  userId: number;
  user: MyUser;
  transporterId: number;
  transporter: PartnerDto;
  // subjectReason: string;
  contactName: string;
  start: Date;
  price: number;
  collect: Date;
  deliver: Date;
  other: Date;
  wasPaid: Date;
  kindTransport: string;
  billingFromId: number;
  billingFrom: BillingFromDto;
  taskOverView: string;
  destinyId: number;
  destiny: DestinyDto;
}
